src/python-common/ceph/deployment/service_spec.py

```diff
@@ -1216,7 +1216,7 @@ def _from_json_impl(cls, json_spec: dict) -> 'SNMPGatewaySpec':
 
     @property
     def ports(self) -> List[int]:
-        return [self.port]
+        return [self.port or 9464]
```

Comment:
Combined with the other changes, does it make to also avoid the explicit definition here and pick it up from the class?

Reply:
right now, this is similar to how we do things in MonitoringSpec